dsfs/dataset.go

```diff
@@ -42,16 +41,16 @@ func LoadDataset(store cafs.Filestore, path string) (*dataset.Dataset, error) {
 func LoadDatasetRefs(store cafs.Filestore, path string) (*dataset.Dataset, error) {
 	ds := dataset.NewDatasetRef(path)
 
-	path = PackageFilepath(store, path, PackageFileDataset)
-	data, err := fileBytes(store.Get(path))
+	pathWithExt := PackageFilepath(store, path, PackageFileDataset)
+	data, err := fileBytes(store.Get(pathWithExt))
 	// if err != nil {
 	// 	return nil, fmt.Errorf("error getting file bytes: %s", err.Error())
 	// }
```

Comment:
Please delete this commented out code while you're here.

Reply:
Part of the reason I kept this in is it relates to the subsequent comment about needing to polls ipfs twice before responding.